### ISSUE TITLE
[Feature Request] Add container support

### ISSUE THREAD

**Issue by Destiny1984** [dimok789/loadiine_gx2#92]
Repacking gamefiles in a container might help with slow loading times and reduce space usage on SD-card (overhead with many small files on large clustersize like 64kb).

Maybe we could reuse the WBFS container, or just use zip files 
(without compression / split to 4gb chunks).

**Comment by dimok789** [comment]
Yeah I already said on the forum that I am thinking of doing that too. Will see about it. Its a bit of work though but doable.

**Comment by mariogamer2** [comment]
It will be good to be compatible with bzip2 and gzip format (more powerfull than zip)
